ql/src/java/org/apache/hadoop/hive/ql/exec/ColumnStatsUpdateTask.java

```diff
@@ -86,8 +86,8 @@ private ColumnStatistics constructColumnStatsFromInput()
               colStats.getStatsDesc().getTableName());
       return colStats;
     }
-    String dbName = work.dbName();
-    String tableName = work.getTableName();
+    final String dbName = work.dbName();
```

Comment:
the "work" should have method(and corresponding field) `getTableName` instead of these 2 getters